core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.scss

```diff
@@ -26,31 +28,53 @@
             font-size: $font-size-sm;
             margin: $spacing-3;
         }
-    }
 
-    .p-dropdown {
-        display: flex;
-        flex-direction: row-reverse; // this allows having the plus icon at the left of the placeholder
-        border: none;
-        background: none;
-        font-size: $spacing-2;
-        margin-left: $spacing-4;
-        padding: $spacing-2;
-    }
-    .p-dropdown:hover {
-        background: $color-palette-primary-200;
-    }
-    .p-dropdown-label.p-inputtext {
-        font-size: $font-size-sm;
-        color: $black;
-        padding: 0 0 0 $spacing-2;
-        max-width: 10rem;
+        .p-dropdown .p-dropdown-trigger {
+            width: 0.875rem; // 14px
+            height: 0.875rem; // 14px
+        }
+
+        .p-dropdown {
+            border-radius: 50%;
+            padding: 0.5625rem; // 9px
+            margin-top: $spacing-3;
+            margin-bottom: $spacing-3;
+        }
     }
-    .p-dropdown-trigger-icon {
-        color: $color-palette-primary;
+}
+
+p-scrollpanel {
+    ::ng-deep {
+        .p-scrollpanel {
+            height: 10.9375rem;
+            width: 100%;
+        }
     }
-    .p-dropdown .p-dropdown-trigger {
-        width: auto;
+}
+
+p-dropdown {
+    ::ng-deep {
+        p-dropdown {
```

Comment:
Should any of this be in the global `CSS` file for `dropdown`?

Reply:
I think this design is specific for this, because It doesn't even look like. a dropdown, is more like a button with the functionality of a dropdown

Reply:
No, the dropdowns here are global